pants

```diff
@@ -2,6 +2,32 @@
 # Copyright 2014 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
+function expected_python_version_to_run() {
+  if [[ "$PANTS_USE_PYTHON3" == 'true' ]]; then
+    echo '3'
+  else
+    echo '2'
+  fi
```

Comment:
Can use the Bash ternary idiom 😎

```bash
major_version="2"; [ "${PANTS_USE_PYTHON3}" = true ] && python_bin_name="3"
echo $major_version
```

Reply:
Tried a twist on that at https://github.com/pantsbuild/pants/pull/7151/files#diff-ca425b88f047ce8ec45ee90e813ada91R7 -- let me know what you think.